dali/test/python/test_operator_readers_nemo_asr.py

```diff
@@ -201,3 +201,16 @@ def test_nemo_asr_reader_alias():
                 new_pipe = nemo_pipe(fn.readers.nemo_asr, [nemo_asr_manifest], read_sr, read_text, dtype, downmix)
                 legacy_pipe = nemo_pipe(fn.nemo_asr_reader, [nemo_asr_manifest], read_sr, read_text, dtype, downmix)
                 compare_pipelines(new_pipe, legacy_pipe, batch_size_alias_test, 50)
+
+
+def test_nemo_asr_reader_pad_last_batch():
+  @pipeline_def(batch_size=128, device_id=0, num_threads=4)
+  def nemo_asr_pad_last_batch_pipe():
+    audio = fn.readers.nemo_asr(manifest_filepaths=[nemo_asr_manifest], pad_last_batch=True,
+                                read_sample_rate = False, read_text = False)
+    return audio
+  pipe = nemo_asr_pad_last_batch_pipe()
+  pipe.build()
+  # Checking that the decoding doesn't fail due to race conditions
+  for _ in range(10):
+    outs = pipe.run()
```

Comment:
Can you check the `outs` result if you really get the desired number of unique samples, and the right sample is repeated?